### DISCUSSION TITLE
Determine ESSID & BSSID

### DISCUSSION THREAD

**Discussion by ravirajbasis** [aircrack-ng/aircrack-ng#2542] [Q&A] [answered]
Hi

How to determine essid & bssid for my network? And how to verify those?

Thanks
rraj

**Comment by Mister-X-** [comment, marked as answer]
SSID is advertised in beacons, in one of the Information Element (IE, aka Tag in Wireshark). BSSID is present in each frame (at least in Management and Data), including beacons, before IE, in what is called the MAC Header (It's one of the 4 addresses).
